### DISCUSSION TITLE
How to play a YouTube playlist

### DISCUSSION THREAD

**Discussion by jannick-vieluf** [jagrosh/MusicBot#672] [Questions] [answered]
Hey,

I think this question is a little bit stupid but I´ve read it is possible to stream a YouTube playlist but I´ve no idea how.
I tried the search and the playlist command with an URL to a YouTube playlist
I hope someone could help me and sorry for this question :D

Jannick

**Comment by YJSoft** [comment, marked as answer]
`!!play (playlist url)` should work fine with latest version.

**Reply by jannick-vieluf** [reply]
Hey,
yes, this works fine, I didn´t try the play command, because I thought this command is for local playlists only.
Maybe is a small command overview a good idea.

Thanks for your help!

**Reply by jagrosh** [reply]
`play playlist <name>` is for local playlists
`play <url>` can play any supported format at that url, including urls to youtube playlists

Typing just the `play` command without any arguments will show you a brief overview.
